### PR TITLE
Allow passing number of master and worker nodes as arguments

### PR DESCRIPTION
**Pull request by swatisehgal**
- This PR allows to parameterise the number of control plane nodes and worker nodes
 - Generates kind config based on the specified number and hence enables creation of
   kind clusters with variable node sizes
 - By default, it continues to create one control-plane node and one worker-node

Signed-off-by: Swati Sehgal <swsehgal@redhat.com>

### CODE REVIEW

File: cluster/kind-replica1.sh [hunk 1]
```diff
@@ -27,9 +29,21 @@ kubeadmConfigPatches:
...
-- role: control-plane
-- role: worker
 EOF
+counter=1
+while [ $counter -le $MASTER_NUM ]
+do
+ echo "- role: control-plane" >> ${CONTEXT}/kindconfig.yaml
+((counter++))
+done
+
```
Comment (ffromani) [inline cluster/kind-replica1.sh:39]: I prefer this idion:

```bash
for idx in $( seq 1 $MASTER_NUM ); do
echo "- role: control-plane" >> ${CONTEXT}/kindconfig.yaml
done
```

but yours works too so let's go ahead

**Review by ffromani — APPROVED** [review]

File: cluster/kind-replica1.sh (hunk 1 above)
Comment (swatisehgal) [reply cluster/kind-replica1.sh:39]: Yeah your suggestion looks much neater! If you want to revert the merge, I can update it as part of this PR itself or submit as another PR? I'll let you decide.

Comment (ffromani) [reply cluster/kind-replica1.sh:39]: no worries, we can improve later. Let's move forward. And thanks for your PRs!

Comment (swatisehgal) [reply cluster/kind-replica1.sh:39]: Ack, Thanks!

**Pull request merged**